src/lib/workers/pdf.ts

```
@@ -164,6 +159,75 @@ export const PDFWorker = {
+          self.Module = Module;
+          import("../ghostscript/gs-worker.js");
+        } else {
+          self.Module["calledRun"] = false;
+          self.Module["postRun"] = Module.postRun;
+          self.Module["preRun"] = Module.preRun;
+          self.Module.callMain();
+        }
+      } catch (error) {
+        reject(error);
+      }
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)